packages/react/src/components/ComponentConformance.test.tsx

```diff
@@ -11,8 +11,9 @@ describe('Top Level Component File Conformance', () => {
   const privateComponents = new Set<string>();
   privateComponents.add('ContextualMenuItemWrapper');
 
+  const componentsBarrelFilesGlob = path.resolve(__dirname, '../..', 'src/components/**/index.ts*');
   const components: string[] = glob
-    .sync(path.resolve(process.cwd(), 'src/components/**/index.ts*'))
```

Comment:
this was tightly coupled to "workspace" execution which started to fail in `rit` context